packages/pipeline/src/scripts/scrape_data.ts

```diff
@@ -293,6 +354,24 @@ if(cli.type === 'events') {
                 console.log(err);
             })
     }
+} else if (cli.type == 'historical_prices') {
+    if (cli.token && cli.from && cli.to) {
+        q.push(_scrapeHistoricalPricesToDB(cli.token, cli.from, cli.to));
+    }
+} else if (cli.type == 'all_historical_prices') {
+    if (cli.from && cli.to) {
```

Comment:
This conditional can be merged with the one above since there is no `else` block.